labelbox/schema/data_row_metadata.py

```diff
@@ -838,7 +838,7 @@ def _validate_parse_number(
 def _validate_parse_datetime(
         field: DataRowMetadataField) -> List[Dict[str, Union[SchemaId, str]]]:
     if isinstance(field.value, str):
-        field.value = datetime.fromisoformat(field.value)
+        field.value = format_iso_from_string(field.value)
```

Comment:
From line 849, we take the `field.value` and convert it into string from the datetime using `format_iso_datetime`. Since `format_iso_datetime` does not take into consideration the timezone, does that mean that the timezone information will get lost here?

For instance, if user inputs `2011-11-04T00:05:23+05:00` as the date metadata, it looks like we'll be upserting `2011-11-04T00:05:23` and the timezone value gets lost? Let me know if I'm not understanding this correctly

Reply:
Time zone does get lost! This is existing before... that was my question - should I fix it?

Reply:
Yes, I know the original issue was to make sure the time string gets properly parsed if timezone passed in.. we should make this also properly parse the timezone, because otherwise, non-UTC timezones will not store accurate datetimes

Reply:
ok... to be clear `format_iso_from_string` is not a problem, as far as I remember from my testing `format_iso_datetime` is the problem, it just formats a string with a `Z` time zone, no conversion. Those two functions are used together in the code (like to validate string is a valid datetime AND convert it back to string)